////////////////////////////////////////////////////////////////////////////////
// Copyright (c) 2017 Dimitry Ishenko
// Contact: dimitry (dot) ishenko (at) (gee) mail (dot) com
//
// Distributed under the GNU GPL license. See the LICENSE.md file for details.

////////////////////////////////////////////////////////////////////////////////
#ifndef GUI_CONTROL_HPP
#define GUI_CONTROL_HPP

////////////////////////////////////////////////////////////////////////////////
#include "src/led.hpp"
#include <QWidget>

////////////////////////////////////////////////////////////////////////////////
namespace gui
{

namespace Ui { class control; }

////////////////////////////////////////////////////////////////////////////////
class control : public QWidget
{
    Q_OBJECT

public:
    explicit control(src::color_led&, QWidget* parent = nullptr);
    ~control();

private:
    Ui::control* ui;
};

////////////////////////////////////////////////////////////////////////////////
}

////////////////////////////////////////////////////////////////////////////////
#endif
